vmdb/db/migrate/20150414204018_create_container_node_identities.rb

```diff
@@ -0,0 +1,16 @@
+class CreateContainerNodeIdentities < ActiveRecord::Migration
+  def change
+    add_column :hardwares, :container_node_id, :bigint
```

Comment:
If we switch to `ContainerNode -> ComputerSystem`, then this relationship should go away/change.